tests/queries/0_stateless/01925_map_populate_series_on_map.sql

```diff
@@ -0,0 +1,35 @@
+drop table if exists map_test;
```

Comment:
btw you can now write `-- { echo }` at the start of the test, and it will echo all queries to the reference, like postgres

Reply:
nice feature, thanks :)